Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/workspaceOptions/AddFolderController.java

```diff
@@ -118,24 +119,22 @@ private void save() throws InterruptedException {
                 return;
             }
 
-            LOGGER.warning("folderName " + folderName);
+            LOGGER.info("Creating folder " + folderName);
             String userId = UserSessionManager.getCurrentUser().getId();
             FolderType currentFolder = this.mainController.getPortalSession().getCurrentFolder();
             int currentParentFolderId = currentFolder == null || currentFolder.getId() == 0 ? 0 : currentFolder.getId();
             this.mainController.getService().createFolder(userId, folderName, currentParentFolderId);
             this.mainController.reloadSummaries();
+
+        } catch (WrongValueException ex) {
+            // Messagebox.show("You have entered invalid value.", "Apromore", Messagebox.OK, Messagebox.ERROR);
```

Comment:
Should this exception also be logged if caught?

Reply:
In this case I was just trying to preserve the original behavior, but that doesn't mean the original behavior was necessarily correct.  Could even be useful for someone interested in usability to log the inputs that users get wrong.  I'll log it at debug level.